app/src/main/java/org/dhis2/usescases/login/LoginActivity.kt

```diff
@@ -66,7 +64,7 @@ class LoginActivity : ActivityGlobalAbstract(), LoginContracts.View {
     private var qrUrl: String? = null
 
     private var testingCredentials: List<TestingCredential> = ArrayList()
-    var userManager : org.dhis2.data.server.UserManager? = null
+    var userManager: org.dhis2.data.server.UserManager? = null
```

Comment:
There is a reason this declaration of the userManager variable has to be explicit?